packages/flutter/lib/src/animation/animations.dart

```diff
@@ -607,6 +607,24 @@ abstract class CompoundAnimation<T> extends Animation<T>
   }
 }
 
+/// An animation of [double]s that tracks the sum of two given animations.
```

Comment:
if we're going to just provide this class, we should change the example at CompoundAnimation to be an example we don't actually provide :-)